packages/preview-middleware-client/test/unit/adp/init.test.ts

```diff
@@ -42,12 +38,56 @@ describe('adp', () => {
         });
     });
 
+    beforeEach(() => {
+        rtaMock.getDefaultPlugins
```

Comment:
Why was this moved from the top, you mock the return value only once. You do not need to mock it again for each test.

Reply:
This is done like that because for each call of init we should have these two specific mock return of getDefaultPlugins